app/src/main/java/space/taran/arknavigator/mvp/model/repo/PlainTagsStorage.kt

```diff
@@ -32,11 +35,10 @@ class PlainTagsStorage
 
     private var lastModified: FileTime = FileTime.fromMillis(0L)
 
-    private val tagsById: MutableMap<ResourceId, Tags> = {
+    private val tagsById: MutableMap<ResourceId, Tags> = runBlocking(Dispatchers.IO) {
```

Comment:
Does `runBlocking` block main thread?

Reply:
Yes, it does. Dispatchers.IO is useless)

It must be replaced with suspend constructor.